.github/workflows/greetings.yml

```
@@ -1,13 +1,32 @@
 name: Greetings
 
-on: [pull_request_target, issues]
+on:
+  pull_request_target:
+    types:
+      - opened
+  issues:
```

It seems that there are various events other than open in the [issue](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#issues).
```suggestion
  issues:
    types:
      - opened
```